docs/reference/commandline/container_prune.md

```diff
@@ -38,6 +40,28 @@ f98f9c2aa1eaf727e4ec9c0283bc7d4aa4762fbdba7f26191f26c97f64090360
 Total reclaimed space: 212 B
 ```
 
+## Filtering
+
+The filtering flag (`-f` or `--filter`) format is of "key=value". If there is more
+than one filter, then pass multiple flags (e.g., `--filter "foo=bar" --filter "bif=baz"`)
+
+The currently supported filters are:
+
+* until (`<timestamp>`) - filter images created before given timestamp
+
+The `until` filter can be Unix timestamps, date formatted
+timestamps, or Go duration strings (e.g. `10m`, `1h30m`) computed
+relative to the daemon machine’s time. Supported formats for date
+formatted time stamps include RFC3339Nano, RFC3339, `2006-01-02T15:04:05`,
+`2006-01-02T15:04:05.999999999`, `2006-01-02Z07:00`, and `2006-01-02`. The local
+timezone on the daemon will be used if you do not provide either a `Z` or a
+`+-00:00` timezone offset at the end of the timestamp.  When providing Unix
+timestamps enter seconds[.nanoseconds], where seconds is the number of seconds
+that have elapsed since January 1, 1970 (midnight UTC/GMT), not counting leap
+seconds (aka Unix epoch or Unix time), and the optional .nanoseconds field is a
+fraction of a second no more than nine digits long.
+
+
```

Comment:
I think we can use one or two examples here;

- an example showing a duration string "The following example prunes all stopped containers that were created more than 30 minutes ago"
- an example showing an absolute date/time
- we may have to emphasize that it's looking at the _creation_ time, not the time that the container was _stopped_ / _exited_ (thinking of this, I _think_ that would actually be more useful; i.e., as a user, I may want to keep containers around that were created 10 months ago, but _exited_ 10 minutes ago. When cleaning up containers, it's more likely I'm no longer interested in a container that exited 2 weeks ago). <-- @vdemeester WDYT?

Reply:
> @thaJeztah @vdemeester The docs have been updated with several examples. Please take a look.
> 
> On a separate note, I agree it probably would be more useful for containers to use "stopped/exited" time.
> 
> The only downside is that, the `stopped/exited` time will only be appropriate for containers. For images or networks, we will not keep records for "deleted images" or "deleted networks". So there might be some inconsistency if we use `stopped/exited` time for containers while at the same time use `created` time for images or networks.
> 
> For that I think it might make sense to keep `until` as `created` time across the board (containers, images, networks), and have another filter for `stopped/exited` time on containers.
> 
> Maybe something like `finished_before` or `running_until`?

last_use?
This could be defined for images, containers and networks.

- container: last time the containers was running (i.e. last time the container stopped).
- image: last time a container based on this image was running.
- network: last time it had a running container in it.

> 
> I can created a separate PR if this is preferable.

Did you ever create this PR?

Reply:
There's this tracking issue to keep track when an image was last "used"; https://github.com/moby/moby/issues/4237

Reply:
Thanks!